Makefile

```diff
@@ -0,0 +1,18 @@
+.DEFAULT_GOAL := help
+
+.PHONY: start
+start: ## Start the development environment (use Docker)
+	$(SUDO) docker run \
+		--rm \
+		-v $(shell pwd):/var/www/FreshRSS:z \
+		-p 8080:80 \
+		--name freshrss \
```

Comment:
Maybe that should either be with a more unique name (in the case the user already has a freshrss instance running), or even no name at all (in this mode without `-d` the command does not return anyway, and deletes the container at the end)

Reply:
I prefer to use a name because it lets us to connect to the container more easily (i.e. without getting the output of `docker ps`) or even to build other tools around the container (e.g. other make commands). What about `freshrss-dev`?

Reply:
Yes, sounds good